src/main/plugin/iso19139.ca.HNAP/layout/config-editor.xml

```diff
@@ -492,21 +492,17 @@
                    in="/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification"
                    or="resourceConstraints"/>
 
-          <field name="temporalRangeSection"
-                 xpath="/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent/gmd:temporalElement"
-                 if="count(gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification) > 0">
-          </field>
-
           <field name="temporalRangeSection"
                  xpath="/gmd:MD_Metadata/gmd:identificationInfo/srv:SV_ServiceIdentification/gmd:extent/gmd:EX_Extent/gmd:temporalElement"
+                 del="../../../.."
                  if="count(gmd:MD_Metadata/gmd:identificationInfo/srv:SV_ServiceIdentification) > 0">
           </field>
 
-          <section name="gmd:EX_Extent">
```

Comment:
I suspect that this is the same as PR #340 and that you only need to add the `displayIfRecord`

Reply:
I'm not sure why it was changed to a field - I guessing that this was done so that the X existed so that it can be removed.

Changing it to a field seems to have cause problems with the non-spatial template which shows the temporary extents twice 

![image](https://github.com/metadata101/iso19139.ca.HNAP/assets/1868233/876ec161-c5a1-498b-84c2-f4e655969a36)

I tried to use the following to exclude the temporal element duplication but that did not work....
xpath="/gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent[not(gmd:temporalElement)]"

I'm thinking that we should be using 
`<section name="gmd:EX_Extent" displayIfRecord="count(gmd:MD_Metadata/gmd:identificationInfo/gmd:MD_DataIdentification/gmd:extent/gmd:EX_Extent) > 0">`

There should be a way to properly remove the section? Just not sure how? This may need more testings....